README.md

```diff
@@ -31,7 +31,7 @@ REQUIRE_VALID_CERTIFICATE=false
 Download the script and deploy the infrastructure:
 
 ```
-DT_AWS_LOG_FORWARDER=https://dynatraceawslogforwarder.s3.amazonaws.com && wget $DT_AWS_LOG_FORWARDER/preview/dynatrace-aws-logs.sh -O dynatrace-aws-logs.sh && chmod +x ./dynatrace-aws-logs.sh && wget $DT_AWS_LOG_FORWARDER/preview/lambda-aws-logs.zip -O lambda-aws-logs.zip && wget $DT_AWS_LOG_FORWARDER/preview/template-aws-logs.yaml -O template-aws-logs.yaml && ./dynatrace-aws-logs.sh deploy --target-url $TARGET_URL --target-api-token $TARGET_API_TOKEN --require-valid-certificate $REQUIRE_VALID_CERTIFICATE
+wget https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/releases/latest/download/dynatrace-aws-log-forwarder.zip && unzip -q dynatrace-aws-log-forwarder.zip && cd dynatrace-aws-log-forwarder && ./dynatrace-aws-logs.sh deploy --target-url $TARGET_URL --target-api-token $TARGET_API_TOKEN --require-valid-certificate $REQUIRE_VALID_CERTIFICATE
```

Comment:
&& cd dynatrace-aws-log-forwarder

Why...
- [ ] One-liner that silently changes current dir is no go for me, let's get rid of this.

Reply:
Yeah, I also don't like it too much but `dynatrace-aws-logs.sh` has hardcoded CF template and Lambda zip paths which it expects in the working dir. Do you know a better way to approach this?

Reply:
Here's a proposal:
https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/pull/12/files